mp/game/momentum/resource/ui/SettingsPanel_OnlineSettings.res

```diff
@@ -79,13 +79,44 @@
 		"Font"		"DefaultVerySmall"
 	}
     
+	"ToggleEntityPanels"
+	{
+		"ControlName"		"CvarToggleCheckButton"
+		"fieldName"		"ToggleEntityPanels"
+		"xpos"		"0"
+		"ypos"		"0"
+        "pin_to_sibling" "ToggleSounds"
+		"pin_corner_to_sibling"		"0"
+		"pin_to_sibling_corner"		"2"
+		"visible"		"1"
+		"enabled"		"1"
+		"tabPosition"		"0"
+		"labelText"		"#MOM_Settings_Entity_Panels"
+		"tooltiptext"		"#MOM_Settings_Entity_Panels_TT"
+		"textAlignment"		"west"
+		"dulltext"		"0"
+		"brighttext"		"0"
+		"wrap"		"0"
+		"centerwrap"		"0"
+		"textinsetx"		"6"
+		"textinsety"		"0"
+		"auto_wide_tocontents"		"1"
+        "auto_tall_tocontents" "1"
+		"use_proportional_insets"		"0"
+		"Default"		"0"
+		"cvar_name"		"mom_entpanels_enable"
+		"cvar_value"		"1"
```

Comment:
Alright, last but not least, remove all of the `cvar_value` keys from this file. This file is old and the previous way to define cvars was through the res file, but defining `cvar_value` makes the game not "save" your settings if the cvar is archived.

Reply:
Ah yeah I see that it is done when making cvars in code. Removed them as requested.